README.md

```diff
@@ -2,79 +2,98 @@
+    > **Note:** Ensure that the Redirect URI value is unique within your domain.  This value can be changed at a later time and does not need to point to a hosted URI.  If the example URI above is already used please choose a unique value.
 
-   >**Important**: Note that in production apps you should always use certificates as your application secrets, but for this sample we will use a simple shared secret password.
+    1. Under **Advanced settings**, set the value of the **Logout URL** to `https://localhost:44334/Account/SignOut`
+    1. Copy the **Redirect URI** as you will need it later.
 
-   d. Under **Platforms**, choose **Add platform**.
+1. Once the app is created, copy the **Application (client) ID** and **Directory (tenant) ID** from the overview page and store it temporarily as you will need both later.
 
-   e. Choose **Web**.
+1. Click **Certificates & secrets** from the current blade navigation pane.
 
-   f. Make sure the **Allow Implicit Flow** check box is selected, and add `https://localhost:44334/signin-oidc` as a **Redirect URL**. This is the base callback URL for this sample.
+    1. Click **New client secret**.
+    1. On the **Add a client secret** dialog, specify the following values:
 
-   >The **Allow Implicit Flow** option enables the hybrid flow. During authentication, this enables the app to receive both sign-in info (the id_token) and artifacts (in this case, an authorization code) that the app can use to obtain an access token.
+        - **Description** = MyAppSecret1
+        - **Expires** = In 1 year
 
-   g. Enter `https://localhost:44334/Account/SignOut` as the **Logout URL**.
-  
-   h. Click **Save**.
+    1. Click **Add**.
 
-   >You'll use the application ID and secret to configure the app in Visual Studio.
+    1. After the screen has updated with the newly created client secret copy the **VALUE** of the client secret and store it temporarily as you will need it later.
 
-4. Configure Permissions for your application. **(Optional)**
+        > **Important:** This secret string is never shown again, so make sure you copy it now.
+        > In production apps you should always use certificates as your application secrets, but for this sample we will use a simple shared secret password.
+1. Click **Authentication** from the current blade navigation pane.
+    1. Select 'ID tokens'
 
-   >Note that we are not required to add permissions for reading user data and sending emails during the app registration as you would do with the [v1 endpoint (ADAL)](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-integrating-applications). The [Incremental and dynamic consent](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-v2-compare#incremental-and-dynamic-consent) capability of the v2 endpoint (MSAL) has made this step optional.
```

Comment:
Was this removed intentionally as not relevant anymore or just by a mistake? I think this is a very valuable information for v1-to-v2 switchers.

Reply:
yes, didn't intent to remove it. Checked with Philipe and will add the revised text.

Reply:
@madansr7 did you get a chance to add the note for this PR?

Reply:
Thank you @madansr7!